lib/assesments/assessment_2/fi_a2_bilal0x01/screens/home/widgets/story_carousel_bilal0x01.dart

```diff
@@ -0,0 +1,35 @@
+part of '../home_screen_bilal0x01.dart';
+
+class _StoryCarouselBilal0x01 extends StatelessWidget {
+  const _StoryCarouselBilal0x01({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final List<String> _carouselImages = const [
```

Comment:
instead of defining and initializing inside build method, try doing it outside, if you're doing it inside build than the leading underscore is not requiered, since this will be a local var